### PR TITLE
Website redesign

### PR DESCRIPTION
**Pull request by ajstrongdev**
- upd(*): Minor website redesigns + Add bug tracker / better news feed
- add blobs
- blur blobs
- Update style.css
- revert blobs
- make header awesome
- update the news feed page
- upd style for feed
- wiki
- footer
- fix box
- New homescreen yay
- Fix: website scaling

### CODE REVIEW

**Comment by ajstrongdev** [comment]
@oklopfer Review this, Don't merge this time :rofl:

File: index.html [hunk 1]
```diff
@@ -47,55 +44,85 @@ <h1>An Ubuntu-based rolling release distribution with sane defaults</h1>
...
+                <img src="assets/images/rhino-setup.png" class="mob-hide" alt="Rhino Setup" width="100%"/>
+            </div>
+        </div>
 
-        <p>
-            We use sane defaults. The XFCE Desktop environment is used for its stable and rock-solid base. Pacstall, our package manager of choice will always provide the latest software, even software that is not available in the Ubuntu repositories, and our custom XFCE configuration provides a traditional desktop that just works, so you can instantly begin using your computer.
-        </p>
+        <div class="sbs-cont">
+            <div class="image">
+                <img src="assets/images/rhino-pkg.png" class="mob-show" alt="Rhino Setup" width="100%"/>
```
Comment (oklopfer) [inline index.html:105]: wrong alt

**Review by oklopfer — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (ajstrongdev) [reply index.html:105]: Wdym

File: index.html [hunk 2]
```diff
@@ -47,55 +44,85 @@ <h1>An Ubuntu-based rolling release distribution with sane defaults</h1>
...
+                <img src="assets/images/rhino-setup.png" class="mob-show" alt="Rhino Setup" width="100%"/>
+            </div>
+            <div class="text">
+                <h1>You're in control</h1>
+                <p>
+                    From the moment you boot up your system, YOU are in control. <code>rhino-setup</code> will greet you upon your first boot, giving you the ability to install additional package managers such as Flatpak, Snap and AppImage. Furthermore, you will be given the options to install additional software to improve your overall experience such as Nala.
+                </p>
+        
+                <p>
+                    Finally, we don’t do any mandatory data collection. You can opt-in with rhino-setup to having apport installed on your system, to send crash and bug reports directly to developers.
```
[the remark this answers is not in the document]
Comment (ajstrongdev) [reply index.html:95]: It makes more sense to keep it under the "user have control" section

File: index.html (hunk 1 above)
Comment (oklopfer) [reply index.html:105]: You set a bunch of images with the alt "Rhino Setup", even though only one of them is.

File: index.html [hunk 3]
```diff
@@ -47,55 +44,85 @@ <h1>An Ubuntu-based rolling release distribution with sane defaults</h1>
...
-            <br>
-            <a href="unicorn/">Learn more</a>
+        <p>We use sane defaults. The XFCE Desktop environment is used for its stable and rock-solid base. Pacstall, our package manager of choice will always provide the latest software, even software that is not available in the Ubuntu repositories, and our custom XFCE configuration, Unicorn, provides a unique, but traditional desktop that just works, so you can instantly begin using your computer.
         </p>
+    </div>
 
+        <div class="h-container">
+        <div class="sbs-cont">
+            <div class="image">
+                <img src="assets/images/rhino-linux.png" class="mob-show" alt="Rhino Setup" width="100%"/>
```
[the remark this answers is not in the document]
Comment (oklopfer) [reply index.html:66]: This is a bad solution. You should just be using css's screen size limits.

File: index.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (oklopfer) [reply index.html:95]: Maybe, but not with the way it's worded, and it also breaks how the rest of the page looks. It should either be rewritten and incorporated into the previous paragraph, or be put as it's own line at the end.

File: index.html (hunk 1 above)
Comment (ajstrongdev) [reply index.html:105]: I see.

File: index.html (hunk 3 above)
[the remark this answers is not in the document]
Comment (ajstrongdev) [reply index.html:66]: Got it, will fix later

File: index.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (ajstrongdev) [reply index.html:95]: I disagree and think it's fine as-is personally.

[the remark this answers is not in the document]
Comment (oklopfer) [reply index.html:95]: ill concede on this one if you fix the other 2

File: assets/css/style.css [hunk 4]
```diff
@@ -197,6 +290,51 @@ body {
...
+
+img {
+  max-width: 100%;
+  max-height:100%;
+}
+
+.text {
+  margin-right: 4em;
+  flex: 40%;
+}
```
Comment (ajstrongdev) [inline assets/css/style.css:335]: Headers are correct size but they now overlap when window size is full-screen

**Comment by ajstrongdev** [comment]
The non-mandatory data collection text needs to be centred

The text for why rhino Linux ideally should be at 60vw, with the images and side-by-side text being 90vw.

**Review by oklopfer — APPROVED** [review]

**Comment by oklopfer** [comment]
closing in favor of https://github.com/rhino-linux/website/pull/27

**Pull request closed without merging**